### PR TITLE
chore: update search page LESQ-470

### PR DESCRIPTION
**Pull request by k-huggs**
## Description

Music year: 1994

- Swaps search results and filters
- changes text on the filters

## Issue(s)

Fixes #LESQ-470

## Acceptance Criteria:

- [x]  The filters should be on the right hand side of the search results
- [x]  Update typography as per Figma

## How to test

1. Go to https://deploy-preview-2110--oak-web-application.netlify.thenational.academy

## Screenshots

How it used to look (delete if n/a):

Screenshot 2023-11-23 at 17 27 02

How it should now look:

Screenshot 2023-11-23 at 17 26 29

## Checklist

- [ ] Added or updated tests where appropriate
- [ ] Manually tested across browsers / devices
- [ ] Considered impact on accessibility
- [ ] Design sign-off
- [ ] Approved by product owner
- [ ] Does this PR update a package with a breaking change

### CODE REVIEW

File: src/components/Checkbox/Checkbox.tsx [hunk 1]
```diff
@@ -42,7 +44,9 @@ export type CheckboxProps = {
   labelFontWeight?: 400 | 600;
   slim?: boolean;
   zIndex?: ZIndex;
-} & SpacingProps;
+  // font?: FontVariant;
```
Comment (njblore) [inline src/components/Checkbox/Checkbox.tsx:47]: 👁️

**Review by njblore — CHANGES_REQUESTED** [review]

File: src/components/Checkbox/Checkbox.tsx (hunk 1 above)
Comment (k-huggs) [reply src/components/Checkbox/Checkbox.tsx:47]: 🥲

**Review by jack-skerrett-oak — APPROVED** [review]

File: src/components/Checkbox/Checkbox.tsx [hunk 2]
```diff
@@ -12,6 +12,8 @@ import getFontFamily from "@/styles/themeHelpers/getFontFamily";
 import FocusUnderline from "@/components/OakLink/FocusUnderline";
 import FieldError from "@/components/FormFields/FieldError";
 import { ZIndex } from "@/styles/utils/zIndex";
+// import { TypographyProps } from "@/styles/utils/typography";
```
Comment (njblore) [inline src/components/Checkbox/Checkbox.tsx:15]: 👁️

File: src/components/Checkbox/Checkbox.tsx [hunk 3]
```diff
@@ -43,6 +45,7 @@ export type CheckboxProps = {
   slim?: boolean;
   zIndex?: ZIndex;
 } & SpacingProps;
+// TypographyProps;
```
Comment (njblore) [inline src/components/Checkbox/Checkbox.tsx:48]: 👁️

File: src/components/Checkbox/Checkbox.tsx [hunk 4]
```diff
@@ -171,6 +174,7 @@ const Checkbox: FC<CheckboxProps> = (props) => {
     labelFontWeight,
     ...spacingProps
   } = props;
+  // const { ...typographyProps } = props;
```
Comment (njblore) [inline src/components/Checkbox/Checkbox.tsx:177]: 👁️

File: src/components/Checkbox/Checkbox.tsx [hunk 5]
```diff
@@ -212,7 +216,10 @@ const Checkbox: FC<CheckboxProps> = (props) => {
 
         {labelText && variant !== "withoutLabel" && (
           <>
-            <CheckboxLabelText fontWeight={labelFontWeight ?? 400}>
+            <CheckboxLabelText
+              // {...typographyProps}
```
Comment (njblore) [inline src/components/Checkbox/Checkbox.tsx:220]: 👁️

File: src/components/SearchFilters/SearchFilterCheckbox.tsx [hunk 6]
```diff
@@ -17,6 +17,7 @@ const SearchFilterCheckbox: FC<SearchFilterCheckboxProps> = (props) => {
   return (
     <Box $width={width}>
       <Checkbox
+        // $font={"heading-light-7"}
```
Comment (njblore) [inline src/components/SearchFilters/SearchFilterCheckbox.tsx:20]: 👁️

**Review by njblore — APPROVED** [review]
approving assuming the comments are removed!

**Pull request merged**